routing.go

```diff
@@ -237,13 +238,13 @@ func NewLimitsResourceRoutes(controller *limitsController.LimitsController) []*r
 	}
 }
 
-func NewTenantsResourceRoutes(controller *tenantsController.Controller) []*rest.Route {
-
+func TenantRoutes(controller *tenantsController.Controller) []*rest.Route {
 	if controller == nil {
 		return []*rest.Route{}
 	}
 
 	return []*rest.Route{
 		rest.Post(ApiUrlInternal+"/tenants", controller.ProvisionTenantsHandler),
+		rest.Get(ApiUrlInternal+"/:tenant/deployments", controller.DeploymentsPerTenantHandler),
```

Comment:
route looks a bit off, should that be something like:

internalAPI `/tenants/:tenant/deployments`